source/apiVolontaria/apiVolontaria/tests/tests_view_UsersId.py

```diff
@@ -96,3 +96,41 @@ def test_retrieve_user(self):
 
         # Check the status code
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_retrieve_profile(self):
+        """
+        Ensure we can retrieve our profile without id.
+        """
+        self.client.force_authenticate(user=self.user)
+
+        response = self.client.get(
+            reverse(
+                'profile',
+            )
+        )
+
+        content = json.loads(response.content)
+
+        # Check id of the user
+        self.assertEqual(content['id'], 1)
+
+        # Check the system doesn't return attributes not expected
+        attributes = ['id', 'username', 'email', 'first_name',
+                      'last_name', 'is_active']
```

Comment:
Shouldn't there be more fields such as `mobile` and `phone` ?

Reply:
Hum... It's strange but yeah i'm ok with you. I'll take a look at that!

Reply:
Notify me when you're done with the test!

Reply:
Oh ok i know why! 

`Phone` and `Mobile` are not merged for the moment. This modification need to be done on the other PR :)

If the PR of mobile and phone is merged before this one, test will fail :)

Reply:
So actually, problem solve on this branch!